.github/workflows/comment-on-e2e-success.yml

```diff
@@ -14,10 +14,12 @@ jobs:
           issue-number: ${{ github.event.client_payload.pr_number }}
           body: |
             E2E Tests Passed
+            Run: https://github.com/gnosis/safe-react-e2e-tests/runs/${{ github.event.client_payload.run_number}}
       - name: Create failure comment
         if: ${{ github.event.client_payload.status == 'failure'}}
         uses: peter-evans/create-or-update-comment@v1
         with:
           issue-number: ${{ github.event.client_payload.pr_number }}
           body: |
-            E2E Tests Failed
\ No newline at end of file
+            E2E Tests Failed
+            Run: https://github.com/gnosis/safe-react-e2e-tests/runs/${{ github.event.client_payload.run_number}}
```

Comment:
`${{ github.event.client_payload.run_number}}` -> `${{ github.event.client_payload.run_number }}`
(a missing space before the closing `}}`)
Could you also fix it in other places in this file?